packages/graph-explorer/src/connector/openCypher/queries/fetchSchema.ts

```diff
@@ -122,7 +128,10 @@ const fetchEdgeLabels = async (
   const values = data.results;
   const labelsWithCounts: Record<string, number> = {};
   for (let i = 0; i < values.length; i += 1) {
-    labelsWithCounts[values[i].label[0] as string] = (values[i].count as number);
+    const label = Array.isArray(values[i].label)
+      ? values[i].label[0]
```

Comment:
nit: Any checks for errors / inconsistent data:
- values[i]is undefined?
- same for undefined: values[i].label

Reply:
Covered, and then some.